client/src/components/loginViewContainer.js

```diff
@@ -1,5 +1,6 @@
 import React from "react";
-const Login = () => {
-  return <div>Login</div>;
+import Login from "./LoginComponent/login";
+const rendering = () => {
+  return <Login />;
 };
-export default Login;
+export default rendering;
```

Comment:
To save some time, this should be changed to something more meaningful
`rendering` is too generic, even though you import it as `loginViewContainer`